generators/python-v2/ast/src/StarImport.ts

```diff
@@ -0,0 +1,18 @@
+import { Writer } from "./core/Writer";
+import { Reference } from "./Reference";
+
+export declare namespace StarImport {
+    interface Args extends Pick<Reference.Args, "modulePath"> {}
+}
+
+export class StarImport extends Reference {
+    public readonly docs: string | undefined;
+
+    public constructor({ modulePath }: StarImport.Args) {
+        super({ name: "*", modulePath });
+    }
+
+    public write(writer: Writer): void {
+        throw new Error("Not intended to be written outside the context of a PythonFile.");
+    }
```

Comment:
nit: Hmm.. not sure whether we should explicitly throw or just no-op here. If you'd rather throw for now to prevent buggy implementations, do you mind updating the message a bit so it's more clear this is coming from StarImport?

```suggestion
    public write(writer: Writer): void {
        throw new Error("Internal error; StarImport node cannot be written outside of PythonFile.");
    }
```

Reply:
I’m down to no-op